awxkit/awxkit/api/pages/credentials.py

```diff
@@ -323,7 +323,7 @@ def create(
         return self.update_identity(
             Credentials(
                 self.connection)).post(payload)
- 
```

Comment:
I can drop these changes, my edit got the remove empty spaces at the end of the line when saving.